### DISCUSSION TITLE
구글 코랩이 안됩니다

### DISCUSSION THREAD

**Discussion by Kimjin201** [openai/whisper#2445] [General]
영상을 업로드하면 에러가 나요 수정 부탁드립니다

**Comment by ryanheise** [comment]
파일을 업로드 하려고 하는 것으로 보아, 여기 GitHub에서 Whisper가 아니라 OpenAI API를 사용하고 계신 것 같습니다. OpenAI API 포럼에서 질문 해 보는 것을 추천합니다:

https://community.openai.com/
